boards/arm/nrf5340_pca10095/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Kconfig - nRF5340 PCA10095 board configuration
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF5340_PCA10095_CPU0 || BOARD_NRF5340_PCA10095_CPU0NS
+
+endif # BOARD_NRF5340_PCA10095_CPU0 || BOARD_NRF5340_PCA10095_CPU0NS
+
+if BOARD_NRF5340_PCA10095_CPU1
+
+# BT_CTLR depends on BT. When BT is enabled we should default to also
+# enabling the controller.
+config BT_CTLR
+	default y if BT
```

Comment:
Why are you actually doing it here, and not in `Kconfig.defconfig`, like it is done for other boards (well, with the exception of `nrf52810_pca10040`, that's perhaps why)?

Reply:
Yes, I didn't see a consistency here, so felt free to add it here.

Reply:
I won't insist on doing it now. It can be corrected later, together with the same correction for `nrf52810_pca10040`.